packages/@okta/vuepress-site/docs/reference/index.md

```diff
@@ -20,17 +20,17 @@ top_links:
 Details on parameters, requests, and rsponses for Okta's API endpoints.
 
 
-Okta APIs generally fall two categories: signing in your users, and managing
-resources in your kta org.
+Okta APIs generally fall into one of two categories: signing in your users or managing
+resources in your Okta org.
 
-## Sign in Your Users
+## Signing in Your Users
```

Comment:
The heading style for documentation is to avoid using gerunds (verbs ending in -ing) in titles and headings. Use sentence case for all headings. 

```suggestion
## Sign in your users
```

Reply:
Changed as requested.